dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -279,11 +294,24 @@ public String getAllelesInForwardStrand() {
                 ^ this.getContigOrientation().equals(Orientation.FORWARD);
 
         String alleles = this.getAlleles();
-        if (forward) {
-            return alleles;
-        } else {
-            return calculateReverseComplement(alleles);
+        if (!forward) {
```

Comment:
I tend to prefer a ternary operator in cases like this, but I don't have a strong opinion about it.